providers/src/airflow/providers/cncf/kubernetes/cli/kubernetes_command.py

```diff
@@ -36,15 +36,20 @@
 from airflow.utils.cli import get_dag
 from airflow.utils.providers_configuration_loader import providers_configuration_loaded
 
+from tests_common.test_utils.compat import AIRFLOW_V_3_0_PLUS
```

Comment:
This shouldn’t import from tests_common.

Reply:
It’s probably easiest to use `getattr` with `get_logical_date_key` in `backcompat`.

Reply:
I have changed it to: 
```
AIRFLOW_VERSION = Version(airflow_version)
AIRFLOW_V_3_0_PLUS = Version(AIRFLOW_VERSION.base_version) >= Version("3.0.0")
```